services/front/package.json

```diff
@@ -5,18 +5,18 @@
     "description": "ZKorum frontend",
     "scripts": {
         "dev": "vite",
-        "dev:force": "vite --force",
-        "build": "tsc && vite build",
-        "build:staging1": "tsc && NODE_ENV=development vite build --mode staging1",
+        "build:staging1": "tsc && NODE_ENV=production vite build --mode staging1",
+        "dev:force": "NODE_ENV='development' && vite --force",
+        "build": "NODE_ENV='production' && tsc && vite build",
```

Comment:
rollback changes:
- NODE_ENV=development or production is set automatically when using `vite` or `vite build`